packages/tspd/src/gen-extern-signatures/gen-extern-signatures.ts

```diff
@@ -64,11 +73,39 @@ export async function generateExternSignatures(
   return diagnostics.diagnostics;
 }
 
+async function getDecoratorLocations(
```

Comment:
do we really need this one, isn't it already getting the same info when compiling in `generateExternSignatureForExports` for that export and we know where that decorator was defined?